app/views/credits/credits_admin_user.ejs

```diff
@@ -32,15 +32,15 @@ this license in a file with the distribution.
         <% for (var t of transactions) { %>
           <tr class="transaction">
             <td> <%= t.id %> </td>
-            <td> <%= t.amount.toFixed(2) + 'ℂ' %> </td>
+            <td> <%= t.amount + 'ℂ' %> </td>
             <td> <%= t.created_at %> </td>
             <td> <%= t.description ? t.description : "No description" %> </td>
             <td> <a href="#" onClick="deleteTransaction(<%= t.id %>)" class="button">Delete</a></td>
           </tr>
           <% } %>
         </table>
     <% } else { %>
-      <h5>There are no transactions.</h5>
+      <h5 style="padding-top: 64px;s">There are no transactions.</h5>
```

Comment:
also maybe instead of an inline style, add a rule to a gigs less file (so we arent separating where styles are)